### ISSUE TITLE
Axis Begin at Zero, not working for Radar

### ISSUE THREAD

**Issue by rezaxkarimi** [team-moeller/better-access-charts#100]
Dear
It's a great work.!

"Begin at zero" is not working for Radar chart type, as well Ticks min and max. please fix it.

Thanks 
Reza

**Comment by team-moeller** [comment]
In the current implementation we do only support x- and y-axis.
Support for radial axis has to be added.

**Comment by team-moeller** [comment]
Documentation for chart.js:
https://www.chartjs.org/docs/2.9.4/axes/radial/linear.html#linear-radial-axis

**Comment by team-moeller** [comment]
The idea is to support a radial axis in addition to an x and a y axis.
This can then also be used to configure the axes in radar and polar charts

**Comment by team-moeller** [comment]
Code snippet:
```
scale: {
	ticks: {
		beginAtZero: true,
		min: 1,
		max: 10,
		stepSize: 0.5
	}
},
```

**Comment by rezaxkarimi** [comment]
HiGood to hear from youI really enjoyed from your work Looking forward for more improvements
Good luckReza

Sent from Yahoo Mail on Android 
 
  On Sat, Mar 5, 2022 at 13:19, Thomas ***@***.***> wrote:   


Code snippet:
scale: {
	ticks: {
		beginAtZero: true,
		min: 1,
		max: 10,
		stepSize: 0.5
	}
},

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by team-moeller** [comment]
Published with release 2.20.11
